README.md

```diff
@@ -52,13 +52,24 @@ require one for your use case.
 * [virtualenv](https://pypi.org/project/virtualenv/)
 * [venv](https://docs.python.org/3/library/venv.html)
 
+NOTE: MacOS sometimes has issues with installing since it comes with python2.7 by default.
+
+### via `homebrew`
```

Comment:
What??? We support Homebrew?
Can you provide an example command for this?

```bash
brew install eth-ape
```

or what is it? I did not know this.

Reply:
I don't think that's what was meant, I think he meant you can install `virtualenv` from homebrew

Reply:
Ah, it is a sub-header among other sub-headers related to installing ape. Maybe we should put it above near the other mention of `virtual env` and just say `**NOTE**: You can also use homebrew(link) to install virtual env.`

Reply:
Can we support homebrew in the future?